rpm/qtmozembed-qt5.spec

```diff
@@ -16,6 +16,7 @@ BuildRequires:  pkgconfig(Qt5Network)
 BuildRequires:  pkgconfig(Qt5OpenGL)
 BuildRequires:  pkgconfig(Qt5Quick)
 BuildRequires:  pkgconfig(Qt5QuickTest)
+BuildRequires:  pkgconfig(sailfishsilica) >= 0.24.15
```

Comment:
This dependency should go to the sailfish-components-webview repository due to OBS repository structure. Please leave enablers over here.

This would imply that following should be in the sailfish-components-webview:
```
    notifyColorSchemeChanged();
    connect(Silica::Theme::instance(), &Silica::Theme::colorSchemeChanged,
            this, &QMozEngineSettingsPrivate::notifyColorSchemeChanged);
```

Reply:
Thanks for reviewing, and for pointing this out. I've moved the silica-related parts into the [sailfish-components-webivew PR](https://github.com/sailfishos/sailfish-components-webview/pull/161) following your advice.